apps/astarte_trigger_engine/lib/astarte_trigger_engine/amqp_events_consumer.ex

```diff
@@ -111,18 +111,22 @@ defmodule Astarte.TriggerEngine.AMQPEventsConsumer do
   end
 
   defp connect() do
-    with {:ok, conn} <- Connection.open(Config.amqp_consumer_options()),
+    with {:ok, amqp_consumer_options} <- Config.amqp_consumer_options(),
+         {:ok, conn} <- Connection.open(amqp_consumer_options),
          {:ok, chan} <- Channel.open(conn),
-         :ok <- Exchange.declare(chan, Config.events_exchange_name(), :direct, durable: true),
-         {:ok, _queue} <- Queue.declare(chan, Config.events_queue_name(), durable: true),
+         events_exchange_name <- Config.events_exchange_name!(),
+         events_queue_name <- Config.events_queue_name!(),
+         events_routing_key <- Config.events_routing_key!(),
```

Comment:
If you use bang then use `=` (it's not going to fallthrough anyways and you're introducing an extra case)